core/winnt/inc/TWinNTSystem.h

```diff
@@ -232,8 +232,8 @@ class TWinNTSystem : public TSystem {
    int               GetServiceByName(const char *service) override;
    char              *GetServiceByPort(int port) override;
    int               OpenConnection(const char *server, int port, int tcpwindowsize = -1, const char *protocol = "tcp") override;
-   int               AnnounceTcpService(int port, Bool_t reuse, int backlog, int tcpwindowsize = -1) override;
-   int               AnnounceUdpService(int port, int backlog) override;
+   int               AnnounceTcpService(int port, Bool_t reuse, int backlog, int tcpwindowsize = -1, ESocketBindOption socketBindOption = = ESocketBindOption::kInaddrAny) override;
```

Comment:
```suggestion
   int               AnnounceTcpService(int port, Bool_t reuse, int backlog, int tcpwindowsize = -1, ESocketBindOption socketBindOption = ESocketBindOption::kInaddrAny) override;
```

Reply:
Thanks. Added the test and removed the typo on the file compiled only on windows.